### PR TITLE
fix(typo): Various typo fixes

### PR DESCRIPTION
**Pull request by tibetiroka**
**Bugfix:** An assortment of random typo fixes.

## Fix Details
Some minor stuff in the data files.

## Testing Done
N/A

## Save File
N/A

### CODE REVIEW

File: data/human/culture conversations.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ mission "Sad Gamblers - Syndicate"
 		random < 1
 	on offer
 		conversation
-			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win. It's one or two day's worth of standard pay.`
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success, and your attention is drawn as one of them wins a moderate sum in a 12th division win. It's one or two days' worth of standard pay.`
```
Comment (lumbar527) [inline data/human/culture conversations.txt:37]: Just a note here because of the question above.

**Review by lumbar527 — APPROVED** [review]
Pretty good!

File: data/human/culture conversations.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ mission "Sad Gamblers - Republic"
...
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply data/human/culture conversations.txt:24]: Without a possessive, would you say something like
> one or two hours
or
> one or two hour

I think the first one is correct? If so, it's probably the same with the possessive form.

[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/culture conversations.txt:24]: The pay belongs to two days (a group) so I would assume that the apostrophe goes like `days'`

**Review by Zitchas — APPROVED** [review]
Looks good to me.

**Review by bene-dictator — APPROVED** [review]

**Pull request merged**